resources/definitions/anycubic_i3_mega_s.def.json

```
@@ -115,6 +115,8 @@
         "skirt_gap": { "value": 5.0 },
         "skirt_line_count": { "value": 4 },
 
+		"meshfix_maximum_deviation": { "value": 0.05 },
+		"meshfix_maximum_resolution": { "value": 0.5 },
```

A couple of changes:

- Spaces instead of tabs
- The `meshfix_maximum_resolution` is already inherited as 0.5 from the `fdmprinter.def.json`, so no need to override it here with the same value

```suggestion
        "meshfix_maximum_deviation": { "value": 0.05 },
```